modules/plugin/geopkg/src/main/java/org/geotools/geopkg/GeoPkgDialect.java

```diff
@@ -82,6 +84,18 @@ public GeoPkgDialect(JDBCDataStore dataStore) {
         geomWriterConfig = new GeoPkgGeomWriter.Configuration();
     }
 
+    // The JDBC aggregate functions doesn't support the geopkg DateTime types.
+    //  This is because they are stored as strings in the database instead of real Date types
+    //   and the JDBC aggregate driver does NOT provide the needed conversions.
+    //
+    // since this doesn't work, we don't support it.
```

Comment:
We tend to use javadoc comments, C style comments are okay inside the code block.